tests/System/Console/Hawk/PreludeTests.hs

```diff
@@ -76,12 +75,11 @@ testEval flags expr = test flags expr ""
 -- 
 -- Making sure that we report errors in the user expression, not our wrapper code:
 -- 
--- TODO: why is this test failing?
--- -->>> testEval [] "[[],"
--- --error: Won't compile:
--- --	user expression:1:5:
--- --    parse error (possibly incorrect indentation or mismatched brackets)
--- --*** Exception: ExitFailure 1
+-- >>> testEval [] "[[],"
+-- error: Won't compile:
+-- 	user expression:1:5: error:
```

Comment:
there's a second "error:" now for some reason. The one on line `79` is generated by hawk, but the one on line 80 isn't. Probably due to switching to a more modern version of ghc.